src/clj/game/cards.clj

```diff
@@ -207,6 +207,42 @@
                                                (vals selected-cards)))]
                            (effect-completed state side (make-result eid {:number counter-count :msg msg})))))}))
 
+(defn pick-credit-providing-cards
+  "Similar to pick-virus-counters-to-spend. Works on :recurring and normal credits."
+  ([provider-func] (pick-credit-providing-cards (hash-map) 0 nil provider-func))
+  ([target-count provider-func] (pick-credit-providing-cards (hash-map) 0 target-count provider-func))
+  ([selected-cards counter-count target-count provider-func]
+   {:async true
+    :prompt (str "Select a card with recurring credits ("
+                 counter-count (when (and target-count (pos? target-count))
+                                 (str " of " target-count))
+                 " credits)")
+    :choices {:req #(and (< 0 (get-counters % :recurring)) (.contains (provider-func) %))}
+    :effect (req (add-counter state :runner target :recurring -1)
```

Comment:
You're hardcoding `:runner` here, but this should be side-neutral. No need to have two `pick-credit-providing-cards` functions.

Reply:
Good catch. This was only the first pass though. I want to refactor the whole function anyways and will fix this in that pass!